lib/internal/crypto/util.js

```diff
@@ -517,6 +517,15 @@ function getBlockSize(name) {
   }
 }
 
+function getDigestSizeInBytes(name) {
+  switch (name) {
+    case 'SHA-1': return 20;
+    case 'SHA-256': return 32;
+    case 'SHA-384': return 48;
+    case 'SHA-512': return 64;
```

Comment:
Can you add a default with unreachable (like throw error)

Reply:
It's done the same way as the method above. I don't feel its necessary. Used hashes are verified far before this is ever invoked.